internal/host/plugin/host_catalog_secret.go

```diff
@@ -90,7 +90,13 @@ func (c *HostCatalogSecret) decrypt(ctx context.Context, cipher wrapping.Wrapper
 	return nil
 }
 
-func (c *HostCatalogSecret) insertQuery() (query string, queryValues []interface{}) {
+// TODO: this is necessary as update logic requires a static
```

Comment:
Reference the jira task inside the TODO as suggested to make it obvious that when the jira task is resolved the TODO should be gone.

```suggestion
// TODO(ICU-2835): this is necessary as update logic requires a static
```

Reply:
Is this a convention that should be followed in the future? Just noting as it's mentioned in the comments a few lines down.

Reply:
👍 updated it a couple of days ago, so we should be fine here.